dataplane/linux/endpoint_mgr.go

```diff
@@ -617,6 +674,20 @@ func (m *endpointManager) resolveWorkloadEndpoints() {
 	})
 }
 
+func wlIdToUniqueName(id *proto.WorkloadEndpointID) string {
+	ids := names.WorkloadEndpointIdentifiers{
+		// Passing "default" for the Orchestrator field tells CalculateWorkloadEndpointName
+		// to construct a name based only on the following fields, which is what we want.
```

Comment:
Couldn't you have two clashing WEPs that differ in their orchestrator value?

Reply:
Yes, maybe that's possible.  I guess I could pass `"default"+id.OrchestratorId` here.  Or I could roll my own unique name function here.  WDYT?

Reply:
See next commit.  I've written `wlIdsAscending` instead, which doesn't rely on `WorkloadEndpointIdentifiers` from libcalico-go.